lte/gateway/c/core/oai/tasks/amf/nas5g_message.cpp

```diff
@@ -224,74 +224,75 @@
  ***************************************************************************/
 int nas5g_message_encode(unsigned char* buffer,
                          const amf_nas_message_t* const msg, uint32_t length,
-                         void* security) {
+                         amf_security_context_t* amf_security_context) {
```

Comment:
No change in indentation has been made, only the adjustment in the type to avoid unnecessary cast and increase safeness of code.
The indentation is automatically placed in the devcontainer environment.